File: resolucoes/java/lista01/src/main/java/org/example/Ex010.java
/*Faça um algoritmo que leia a largura e altura de uma parede, calcule e 
mostre a área a ser pintada e a quantidade de tinta necessária para o serviço,
 sabendo que cada litro de tinta pinta uma área de 2metros quadrados. */

package org.example;

import java.util.Scanner;

public class Ex010 {
    public static void main(String[] args) {
        Ex010.main();
    }

    public static void main() {

        Scanner sc = new Scanner(System.in);
        System.out.print("Qual largura da parede?: ");
        Float largura = sc.nextFloat();
        System.out.print("Qual altura da parede?: ");
        Float altura = sc.nextFloat();
        Float area = altura * largura;
        System.out.printf(
                "A área da parede é %sm sabendo que se usa 2 litros será utlizado %s litros de tinta para pinta-lá",
                area,
                area / 2);
        sc.close();

    }
}

File: resolucoes/Lua/ex010.lua
print("Digite a largura da parede em metros:")
largura = tonumber(io.read()) -- Lê a largura da parede e converte para um número

print("Digite a altura da parede em metros:")
altura = tonumber(io.read()) -- Lê a altura da parede e converte para um número

area = largura * altura -- Calcula a área da parede a ser pintada

tinta = area / 2 -- Calcula a quantidade de tinta necessária, sabendo que cada litro de tinta pinta 2 metros quadrados

-- Mostra o resultado na tela
print("A area a ser pintada e de " .. area .. " metros quadrados.")
print("Serao necessarios " .. tinta .. " litros de tinta.")
